engine/src/execution_context.rs

```diff
@@ -29,22 +29,14 @@ impl<'e> ExecutionContext<'e> {
         self.scheme
     }
 
-    pub(crate) fn get_field_value_unchecked(&'e self, field: Field<'e>) -> LhsValue<'e> {
+    pub(crate) fn get_field_value(&'e self, field: Field<'e>) -> Option<LhsValue<'e>> {
```

Comment:
It's still unchecked. As I explained in the past when you first tried to change the name, this is about scheme compatibility, not about the missing fields :)

Reply:
Oh sorry i did not understood that properly.